### ISSUE TITLE
Latest msys2 (cygwin 3.2) breaks windows terminal input on 

### ISSUE THREAD

**Issue by UncleInf** [microsoft/terminal#10091]
### Windows Terminal version (or Windows build number)

10.0.19042.985, 1.8.1032.0

### Other Software

msys2-x86_64-20210419

### Steps to reproduce

1. Open bash from msys2 in windows terminal
2. split pane
3. try to input in previous pane

This issue will be more widespread, when git for windows updates its msys2 to latest and in turn uses cygwin 3.2.

### Expected Behavior

I can enter text into pane

### Actual Behavior

Cannot enter text

**Comment by zadjii-msft** [comment]
Is there a thread tracking this upstream? If msys2 worked in the Terminal _before_ the update, and it doesn't _after_ the update, that would suggest to me that the root cause of the issue doesn't lie with the Terminal, but instead with cygwin/msys2.

**Comment by UncleInf** [comment]
You are right describing the situation - it seems it is cygwins fault

**Comment by zadjii-msft** [comment]
If there's an upstream issue, feel free to link back to it here, so future readers can follow it. Thanks!

**Comment by UncleInf** [comment]
After further investigation I've pinpointed that it only gets stuck with default ohmyzsh theme - robbyrussell. Linking issue raised there: https://github.com/ohmyzsh/ohmyzsh/issues/9953

For reference my stack:
Terminal: windows terminal preview 1.9
Cygwin: CYGWIN_NT-10.0 LAT-5F3FMQ2 3.2.0(0.340/5/3) 2021-03-29 08:42 x86_64 Cygwin
Shell: zsh 5.8 + oh my zsh (706b2f3) + theme robbyrussell (default)
Windows version: 10.0.19043 N/A Build 19043 

@zadjii-msft I've also tracked down that it is the actual windows terminal **window resizing** that makes input stuck. May be this clue brings something to your mind? Coz it might be hard to pint point faulty part at whole "input" stack.
